scalafix-cli/src/main/scala/scalafix/cli/CliRunner.scala

```diff
@@ -315,8 +319,8 @@ object CliRunner {
         .map(path => AbsolutePath(path)(common.workingPath))
         .toList
 
-    def resolveClasspath: Configured[Classpath] =
-      classpath match {
+    def resolveClasspath: Configured[Classpath] = {
+      classDirectory match {
```

Comment:
Not a blocker but I don't like that this name hints that the classpath is a single directory entry, which is not what I usually see for a classpath.